style.css

```diff
@@ -5209,7 +5209,6 @@ a.to-the-top > * {
 		position: static;
 		padding-right: 20px;
 		font-size: 15px;
-		color: #000;
```

Comment:
@ianbelanger79 Doesn't removing this line undo the following requirement pointed out by @andersnoren?

> * When the toggle text is displayed next to the icon, the text should have a font-size of 15px, a right padding of 20px **and the main body color, rather than the light gray one**.

Reply:
It may remove that requirement, but having a set color causes issues when the Header Background color is set to `#000` or any dark color. The toggle text is no longer visible. @andersnoren do you agree?

Reply:
@ianbelanger79 @glauberm Setting it to `color: inherit` here instead will make it inherit the color from the toggle element, which is updated with the color settings.